exampleSite/config.toml

```diff
@@ -54,9 +54,10 @@ copyright = ""            # default: author.name ↓        # 默认为下面配
   debug = false             # If true, load `eruda.min.js`. See https://github.com/liriliri/eruda
 
   since = "2017"            # Site creation time          # 站点建立时间
+
   # use public git repo url to link lastmod git commit, enableGitInfo should be true.
   # 指定 git 仓库地址，可以生成指向最近更新的 git commit 的链接，需要将 enableGitInfo 设置成 true.
-  gitRepo = ""
+  gitCommitPrefix = ""
```

Comment:
Although `gitCommitPrefix` is ok, I prefer using gitRepo in `config.toml`.

1. `gitCommitPrefix`  is a little bit odd, if using `gitCommitPrefix`, it's better to add an example in config comment.
2. remove `gitRepo` directly will break the old versions.

So, if we can implement a method to generate the final commit URL(check GitHub, Bitbucket and so on) will be better?